tic-tac-toe/src/tic_tac_toe/board.clj

```diff
@@ -0,0 +1,79 @@
+(ns tic-tac-toe.board
+  (:require [tic-tac-toe.marks :refer :all]))
+
+(def dimension 3)
+
+(defn- one-based [value]
+  (+ value 1))
+
+(defn- three-matching-player-symbols? [row]
+  (let [[one two three] row]
+    (and (= one two three) (not(= one nil)))))
+
+(defn- winning-row? [rows]
+  (some true? (map
+                (fn[row] (three-matching-player-symbols? row))
```

Comment:
Ahh, I had left the `three-matching-player-symbols?` in brackets - so was executing it too early I think..

I had:
(some true? (map three-matching-player-symbols? rows) ))

I guess I don't need the brackets around the `three-matching...` here because map will execute it.

What you suggested now works. Will add it to the new PR. Cheers!